develop.mjs

```diff
@@ -23,6 +23,11 @@ const run = (command = "") => {
         name: "shop server",
         cwd: path.resolve(__dirname, "shop"),
       },
+      {
+        command,
+        name: "dashboard server",
+        cwd: path.resolve(__dirname, "dashboard"),
+      },
```

Comment:
**Issue**: The new block for the 'dashboard server' lacks error handling which could lead to unhandled promise rejections if the command fails. <br> **Fix**: Implement error handling for the new command block to ensure stability and reliability of the script execution. <br> **Code Suggestion**: 
 ```
 +      {
 +        command,
 +        name: "dashboard server",
 +        cwd: path.resolve(__dirname, "dashboard"),
 +      }.catch(error => console.error('Error starting dashboard server: ${error}'))
 ```